src/renderer/services/chain/fees/unstake.ts

```diff
@@ -0,0 +1,79 @@
+import * as RD from '@devexperts/remote-data-ts'
+import { Chain, BaseAmount } from '@thorchain/asgardex-util'
+import * as FP from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+import * as Rx from 'rxjs'
+import * as RxOp from 'rxjs/operators'
+
+import { BASE_CHAIN } from '../../../const'
+import { isBaseChain } from '../../../helpers/chainHelper'
+import { liveData, LiveData } from '../../../helpers/rx/liveData'
+import { observableState } from '../../../helpers/stateHelper'
+import * as BNB from '../../binance/service'
+import * as BTC from '../../bitcoin/context'
+import { selectedPoolChain$ } from '../../midgard/common'
+import { getCrossChainUnstakeMemo$ } from '../memo'
+import { FeeLD, StakeFeesLD } from '../types'
+import { reloadStakeFeesByChain } from './fees.helper'
+
+const { get$: unstakePercent$, set: updateUnstakePercent } = observableState(0)
+
+// reload fees
+const updateUnstakeFeesEffect$ = Rx.combineLatest([selectedPoolChain$, unstakePercent$]).pipe(
+  RxOp.tap(([oChain, _]) =>
+    FP.pipe(
+      oChain,
+      O.map((chain) => {
+        // reload base-chain
+        reloadStakeFeesByChain(BASE_CHAIN)
+        // For x-chains transfers, load fees for x-chain, too
+        if (!isBaseChain(chain)) reloadStakeFeesByChain(chain)
+        return true
+      })
+    )
+  )
+)
+
+const unstakeFeeByChain$ = (chain: Chain): FeeLD => {
+  switch (chain) {
+    case 'BNB':
+      return FP.pipe(
+        unstakePercent$,
+        RxOp.switchMap(() => BNB.stakeFee$)
+      )
```

Comment:
I would suggest not to depend on changes of unstakePercent$ for BNC fees. BNC fee won't change that often, maybe once a year or so :)
```
return BNB.stakeFee$
```

Reply:
@veado as we need only BASE_CHAIN fee we will not fall into BTC chain which was the only one who demanded percents. i removed percents here for now and now this is just a triggerStream without updating on withdraw percent changes